packages/cli/core.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const { Mixin, strategies } = require('hops-mixin');
+
+module.exports = {
+  Mixin,
+  strategies,
```

Comment:
Should we export `hops-config` as `config` here as well?

Reply:
As discussed, we won't include the `hops-config` for now in the default preset.